Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -665,15 +665,15 @@ - (void)validateExpectedState:(nullable NSDictionary *)expectedState {
             TargetId targetID = [targetIDString intValue];
             ByteString resumeToken = MakeResumeToken(queryData[@"resumeToken"]);
             NSArray *queriesJson = queryData[@"queries"];
-            std::vector<QueryData> queries;
+            std::vector<TargetData> queries;
```

Comment:
Should this be `targets`? Maybe not because the JSON actually contains "queries" in it.

Reply:
Yeah, l'd like to keep using `queries` here.